src/SimpleTraining/src/main/java/cz/neonit/klemsa/training/dao/communication/MessageInfoLoader.java

```diff
@@ -0,0 +1,19 @@
+package cz.neonit.klemsa.training.dao.communication;
+
+import cz.neonit.klemsa.training.domain.communication.CommunicationInfo;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author tomasklemsa
+ */
+public interface MessageInfoLoader {
+
+    /**
+     * Gets all message info on the specified date.
+     * @param date in witch you search for message info.
+     * @return List of message info witch available in current date.
+     */
+    List<CommunicationInfo> getMessagesInfo(Date date);
```

Comment:
Proč je tady použitý objekt Date?
Jak parametr z RESTu, tak infix v názvu soboru jsou reprezentovány pomocí String, tak proč mezi tím převádět tam a zpátky?